ui/app/components/app/home-notification/home-notification.component.js

```diff
@@ -3,6 +3,9 @@ import classnames from 'classnames'
 import { Tooltip as ReactTippy } from 'react-tippy'
 import PropTypes from 'prop-types'
 import Button from '../../ui/button'
+import BoxedMessage from '../../ui/boxed-message'
+import connectIcon from '../../../../../app/images/icons/connect.svg'
```

Comment:
We don't use webpack, so these loaders won't work. They should be referenced using `img` tags instead, or converted to React components..